FFmpegInterop/FFmpegInteropLogging.cpp

```diff
@@ -65,7 +65,7 @@ namespace winrt::FFmpegInterop::implementation
 				line[i] = '\0';
 			}
 
-			FFMPEG_INTEROP_TRACE("%S", line.get());
+			FFMPEG_INTEROP_TRACE("%hs", line.get());
```

Comment:
does the to_hstring on line 57 above support char[]?

Reply:
Yes, there is a templated to_hstring() overload that accepts anything convertible to a std::string_view:

[cppwinrt/strings/base_string.h · microsoft/cppwinrt (github.com)](https://github.com/search?q=repo%3Amicrosoft%2Fcppwinrt%20%22template%20%3Ctypename%20T%2C%20std%3A%3Aenable_if_t%3Cstd%3A%3Ais_convertible_v%3CT%2C%20std%3A%3Astring_view%3E%2C%20int%3E%20%3D%200%3E%22%20%22hstring%20to_hstring(T%20const%26%20value)%22&type=code)
```
template <typename T, std::enable_if_t<std::is_convertible_v<T, std::string_view>, int> = 0>
hstring to_hstring(T const& value)
```

A std::string_view can be constructed from a const char*:

[std::basic_string_view<CharT,Traits>::basic_string_view - cppreference.com](https://en.cppreference.com/w/cpp/string/basic_string_view/basic_string_view)
> `constexpr basic_string_view( const CharT* s );`
>
> Constructs a view of the null-terminated character string pointed to by s, not including the terminating null character. The length of the view is determined as if by Traits::length(s). The behavior is undefined if [s, s + Traits::length(s)) is not a valid range. After construction, [data()](https://en.cppreference.com/w/cpp/string/basic_string_view/data) is equal to s, and [size()](https://en.cppreference.com/w/cpp/string/basic_string_view/size) is equal to Traits::length(s).